src/view.c

```diff
@@ -624,6 +624,16 @@ view_toggle_always_on_top(struct view *view)
 	}
 }
 
+void
+view_move_to_front(struct view *view)
+{
+	assert(view);
+	if (view->impl->move_to_front) {
+		view->impl->move_to_front(view);
+		cursor_update_focus(view->server);
+	}
+}
```

Comment:
The assert here changes the requirements of calling this function, I think all callers do check for `if (view)` so it should be fine but maybe we should document that requirement in the header file.

Side note:
This blurs the line between `desktop.c` and `view.c` a bit, I think usually `desktop.c` handles relations of views with each other, as in this case the stacking order. So I think I'd slightly prefer this kept as `desktop_move_to_front()` or maybe `desktop_move_view_to_front()` or just `desktop_raise_view()`.

Reply:
Have restored `if (!view) { return }`
Have re-instated `desktop_move_to_front()`. Agree - it's more consistent to keep it in `desktop.c`